avocado/plugins/human.py

```diff
@@ -49,7 +49,7 @@ def pre_tests(self, job):
         self.log.info("JOB ID     : %s", job.unique_id)
         replay_source_job = getattr(job.args, "replay_sourcejob", False)
         if replay_source_job:
-            self.log.info("SRC JOB ID : %s", self.__replay_source_job)
+            self.log.info("SRC JOB ID : %s", replay_source_job)
```

Comment:
Good catch. Btw we should fix avocado and stop using global disabled `E1101`, which would have caught this one...